decidim-comments/app/frontend/comments/comment.fragment.graphql

```diff
@@ -1,7 +1,12 @@
 fragment Comment on Comment {
-  body
-  createdAt
-  author {
-    name
+  ...CommentData
```

Comment:
If I'm not wrong, the UI isn't currently limiting this. The fact that we "preload" three levels of depth is just for convenience.

I can see how we'll need a `repliesCount` field in the future that will allow us to load replies asynchronously, so if we decide we want 4 levels of depth on the backend side, nothing would need to be changed here (unless we want to show the 4th level of depth on first page render).